chef/cookbooks/ohai/recipes/default.rb

```diff
@@ -95,4 +95,19 @@
 
 o = Ohai::System.new
 o.all_plugins
+
+# drop virtual interfaces, to not overload chef
+virtual_intfs = ["tap", "qbr", "qvo", "qvb", "brq", "ovs"]
+o.data["network"]["interfaces"].each_key do |intf|
+  if virtual_intfs.include?(intf.slice(0..2))
+    o.data["network"]["interfaces"].delete(intf)
+    o.data["counters"]["network"]["interfaces"].delete(intf)
```

Comment:
Actually, it was some mini-optimization (slightly confusing), but doesn't matter because of... (next comment)